### ISSUE TITLE
Do we have HoTT 5.8.4

### ISSUE THREAD

**Issue by Alizter** [HoTT/HoTT#1036]
It is not in the HoTT book file, but more specifically do we have a lemma saying that if we have the equivalence in 5.8.4 (iv) then we have the contractility in (v)? I couldn't find anything in Types.Sigma or Types.Path...

**Comment by mikeshulman** [comment]
I don't think so, but it should be an easy consequence of `equiv_functor_sigma_id` along with `contr_basedpaths` and `contr_equiv`.
